examples/synthea-breast-cancer/Ali_Laiba_syntheaEDA/SyntheaClassifiers.ipynb

```diff
@@ -0,0 +1,1710 @@
+      "     Stage 3       0.97      1.00      0.98        31\n",
+      "     Stage 4       1.00      1.00      1.00         9\n",
+      "\n",
+      "    accuracy                           0.99       280\n",
+      "   macro avg       0.98      0.99      0.99       280\n",
+      "weighted avg       0.99      0.99      0.99       280\n",
+      "\n",
+      "\n",
+      " Balanced Accuracy  Macro F1 Score      MCC\n",
+      "          0.992857        0.988714 0.976935\n",
+      "\n",
+      "AVERAGE CROSS-VALIDATION SCORE: 0.9621428571428572\n"
+     ]
+    }
+   ],
+   "source": [
+    "pred_mult_OH = mult_OH.predict(X_test_OH)\n",
+    "print_results(y_test_OH, pred_mult_OH)\n",
+    "print_cross_validation(MultinomialNB(), X_OH, y_OH)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Viewing Results for One-Hot Encoded Bernoulli Classifier**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 34,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CONFUSION MATRIX:\n",
+      "[[134   4   2   0]\n",
+      " [  0  96   4   0]\n",
+      " [  0   0  31   0]\n",
+      " [  0   0   0   9]]\n",
+      "\n",
+      "CLASSIFICATION REPORT:\n",
+      "              precision    recall  f1-score   support\n",
+      "\n",
+      "     Stage 1       1.00      0.96      0.98       140\n",
+      "     Stage 2       0.96      0.96      0.96       100\n",
+      "     Stage 3       0.84      1.00      0.91        31\n",
+      "     Stage 4       1.00      1.00      1.00         9\n",
+      "\n",
+      "    accuracy                           0.96       280\n",
+      "   macro avg       0.95      0.98      0.96       280\n",
+      "weighted avg       0.97      0.96      0.96       280\n",
+      "\n",
+      "\n",
+      " Balanced Accuracy  Macro F1 Score      MCC\n",
+      "          0.979286        0.962467 0.942941\n",
+      "\n",
+      "AVERAGE CROSS-VALIDATION SCORE: 0.9571428571428571\n"
+     ]
+    }
+   ],
+   "source": [
+    "pred_bern_OH = bern_OH.predict(X_test_OH)\n",
+    "print_results(y_test_OH, pred_bern_OH)\n",
+    "print_cross_validation(BernoulliNB(), X_OH, y_OH)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Analysing One-Hot Encoded Results**\n",
+    "\n",
+    "From the results of the two One-Hot Encoded Classifiers, we can see that the one-hot encoding made a significant impact as predicted. It essentially converts all categorical data into binary data, allowing for proper classification by the Bernoulli classifier, and given that each of the categories no have their own label, it also acts as a frequency counter given that each category that appears will have a frequency of 1 compared to categories that don't appear and hence have a frequency of 0. Thus both the Bernoulli and Multinomial classifiers perform much better with this dataset. \n",
```

Comment:
Super cool, thanks for putting in the effort to one-hot encode everything for this experiment